tests/test_config_templates/test_git_repo_fixture.py

```
@@ -0,0 +1,138 @@
+        assert (scripts_dir / "bump.sh").exists()
+
+    def test_git_repo_scripts_are_executable(self, git_repo):
+        """GitHub scripts should be executable."""
+        for script in ["release.sh", "bump.sh"]:
+            script_path = git_repo / ".github" / "scripts" / script
+            assert os.access(script_path, os.X_OK), f"{script} is not executable"
+
+    def test_git_repo_is_initialized(self, git_repo):
+        """Git repo should be properly initialized."""
+        result = subprocess.run(
+            ["git", "rev-parse", "--git-dir"],
```

Ruff (0.14.8)

Starting a process with a partial executable path

(S607)